lib/ansible/modules/network/f5/bigip_firewall_dos_profile.py

```diff
@@ -73,32 +73,6 @@
       server: lb.mydomain.com
       user: admin
   delegate_to: localhost
-
```

Comment:
Why are you removing this doc?

P.S. It's harder to evaluate PRs, which touch unrelated things, please consider having separated atomic PRs in future.